docs/tutorials/field_type/7_add_content_and_edit_views.md

```diff
@@ -1,275 +1,116 @@
-        $config = Yaml::parse(file_get_contents($configFilePath));
-        $container->prependExtensionConfig($extensionName, $config);
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $label = $fieldDefinition->getName($formConfig->getOption('mainLanguageCode')) ?: reset($names);
+        $fieldType = $this->fieldTypeService->getFieldType($fieldDefinition->fieldTypeIdentifier);
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        TextType::class,
+                        [
+                            'required' => false,
+                            'label' => $label
+                        ]
+                    )
+                    ->setAutoInitialize(false)
+                    ->getForm()
+            );
     }
 }
 ```
 
-Remember to dump your assets before trying to run this code for the first time.
-After this is done, we should be able to edit the Content Item containing our Tweet Field Type.
-
-## Implement content view
-
-The steps needed to create content view are analogical to the ones needed to create edit view.
-
-### Template file
-
-First, we need to add a template file that will be responsible for displaying the content of our Field Type:
-
-```html
-//Resources/public/templates/fields/view/tweet.hbt
+Next thing is to register the FormMapper as a service, so the system would know to use it to automatically add the input field to the Content Type edit form. You can read more about services and service container in the documentation: https://doc.ezplatform.com/en/latest/guide/service_container/. To register the FormMapper as a service, let's add the following lines to `fieldtypes.yml`:
+``` yml
+// Resources/config/fieldtypes.yml
 
-<div class="ez-fieldview-row pure-g">
-    <div class="ez-fieldview-label pure-u">
-        <p class="ez-fieldview-name"><strong>{{ translate_property fieldDefinition.names }}</strong></p>
-    </div>
-    <div class="ez-fieldview-value pure-u"><div class="ez-fieldview-value-content">
-        {{#if isEmpty }}
-            {{translate 'fieldview.field.empty' 'fieldview'}}
-        {{else}}
-            {{ value.url }}
-        {{/if}}
-    </div></div>
-</div>
+    ezsystems.tweetbundle.fieldtype.eztweet.form_mapper:
+        class: EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet\FormMapper
+        tags:
+            - {name: ez.fieldFormMapper.value, fieldType: eztweet}
+        arguments: ['@ezpublish.api.service.field_type']
 ```
 
-### Javascript view file
+# DataTransformer
 
-Next, we need to implement Javascript file that will be responsible for handling the template:
+As mentioned earlier we also somehow need to tell our FormMapper how to transform ValueObject into single string which contains URL to given tweet. We do this by creating implementation of DataTransformerInterface.
```

Comment:
`As mentioned earlier, we also need to tell` ... `into a single string which contains the URL to a given tweet` ... `creating an implementation`